src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -0,0 +1,135 @@
+import { useBackend } from '../../contexts/BackendContext';
+import { useState } from 'react';
+import { Flex, Text, Textarea, Stack, Box, Spacer, Button, ButtonGroup, Checkbox } from '@chakra-ui/react';
+
+// All in Flexbox component
+/* Supply Request
+Checkbox for checkboxes duh
+Textarea for notes section
+Button for submit
+*/
+/* FPH Info
+Box
+*/
+
+// notificationRouter.post('/', async (req, res) => {
+//   const { business_id: businessId, message, timestamp, been_dismissed: beenDismissed } = req.body;
+//   try {
+//     await db.query(
+//       `
+//         INSERT INTO notification (business_id, message, timestamp, been_dismissed)
+//         VALUES
+//         ($(businessId), $(message), $(timestamp), $(beenDismissed));
+//       `,
+//       { businessId, message, timestamp, beenDismissed },
+//     );
+//     res.status(200).json({
+//       status: 'Success',
+//     });
+//   } catch (err) {
+//     res.status(500).send(err.message);
+//   }
+// });
+
+
+
+
+const ContactUs = () => {
+  const { backend } = useBackend();
+  const [ID, setID] = useState(0);
```

Comment:
Since FPH business_id is always zero, we don't need a react state to set it, instead you can just make a const statement that sets the FPH business_id to zero.